data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
+	description `Aurelis, is a mineral rich world that is stuck in an almost permanent winter. Its oceans and hills are covered by ice throughout most of the year. Despite all the snow and ice, it carries a sizeable population heavily engaged in mining operations.`
+	description `	The Alteran Military also has a strong presence on Aurelis. Member of the local population are usually quite strong, since they are not only miners, but also live under higher gravity than most Alterans. The Verangian Guard, an elite Alteran army division, trains here and most of its members are recruited locals.`
+	spaceport `Landing in the Aurelis spaceport is actually quite difficult, the port itself isn't too big or busy, but the low temperatures and the regular snowfall cause the spaceport to be buried under snow for most of the time. The spaceport agency on Aurelis is unable to keep up with the snow during storms and their immediate aftermath. A newer section for the spaceport with floor heating has been built and tested, but no decision has been met yet, as the maintenance costs are quite high.`
 	shipyard "Ancient Advanced"
 	outfitter "Ancient Advanced"
 
 planet "Avalon Station"
 	landscape land/sea17-harro
-	description `Just from looking at its exterior, it is obvious that this is a finished Quarg ringworld, though it doesn’t seem to be in the posession of the Quarg anymore. While it looks just like any other ring from outside, once your inside, you can’t get your head around the thought that this ring must have been taken over by the Alterans ages ago, the inside barely resembles its original form, which considering the time taken to build such rings, must have taken a long time as well. Large portions of the Quarg interior and technology have been augmented or replaced with Alteran designs and the gravitational force is quite a bit stronger, even compared to human standards.`
-	spaceport `The spaceport seems to have been replaced almost completely as well and is much more open than the original Quarg design, as the Alterans have nothing to hide and even the military section is only partially off-limits to civilians. In order to travel around the spaceport, or rather the entire ringworld, a system of local teleporters and magnetically levitated trains has been built.`
-	spaceport `	The experience on this ring is nothing like that of a Quarg ring, it seems busier and much more welcoming, stemming mostly from the more open Alteran design, but the people and talking present. Several shipyards and outfitters can be found around the spaceport and a nice bar has placed itself right next to one of those teleporters.`
```

Comment:
The first sentence I had really difficulties understanding. I rewrote it with what I understood, but it could be completely off. In the seconds sentence, bars usually do not place themselves, the owner does. But writing it that way would have been too long.